packages/modular-scripts/src/__tests__/index.test.ts

```diff
@@ -189,6 +178,28 @@ describe('modular-scripts', () => {
   });
 
   it('can start an app', async () => {
```

Comment:
Sigh.

Reply:
Oh the problems of buying a new MacBook. 😄